### DISCUSSION TITLE
Troubleshooting: Can't install the addon

### DISCUSSION THREAD

**Discussion by BlenderDefender** [PidgeonTools/SuperProjectManager#21] [General]
# Troubleshooting Guide:
#### ModuleNotFoundError: No module named 'blender_project_manager-1'
![Module Error in line 351](https://user-images.githubusercontent.com/38502179/108857793-5b9d6600-75e3-11eb-9f4c-64e28144bd67.png)

This is an issue many people have experienced and reported so far. When trying to install the addon, this message appears:
Traceback (most recent call last):
File "C:\3D\Blender Launcher\stable\blender-2.91.2-windows64\2.91\scripts\modules\addon_utils.py", line 351, in enable
mod = import(module_name)
ModuleNotFoundError: No module named 'blender_project_manager-1'

Fix: The .zip File or the folder within it has somehow been renamed to "blender_project_manager-1". Somehow, if there's a "-NUMBER" in an addons folder name, Blender searches for a non-existent python module.

If you didn't understand this, no problem:
1. Unzip the folder
2. Rename the extracted folder to 'blender_project_manager'
3. zip the folder
4. Install the .zip file you've just created.

**Comment by Larry3d** [comment]
rename Folder from C:\Users\XXXXX\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\blender_project_manager-1.2.0
to C:\Users\XXXXX\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\blender_project_manager

the -1 in the folder is the problem.

**Reply by BlenderDefender** [reply]
Exactly, nice it worked! Somehow, if theres a "-number" in an addons folder name, Blender searches for a python module.

**Reply by BlenderDefender** [reply]
btw.: This **C:\Users\XXXXX\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons** can be written as this: **%APPDATA%\Blender Foundation\Blender\2.92\scripts\addons**

It's shorter and it's even a valid path for any Windows Computer (with Blender 2.92 installed)
